src/toil/jobStores/utils.py

```diff
@@ -209,15 +218,24 @@ def _writer(self):
             if e.errno != errno.EPIPE:
                 raise
 
-    def __init__(self):
+    def __init__(self, encoding=None, errors=None):
+        """
+        :param str encoding: the name of the encoding used to encode the file. Encodings are the same
```

Comment:
You should make it clear that these apply to whoever reads the pipe, and that the `writeTo` implementation still needs to work on binary streams.

Reply:
Also I noticed the `writeTo` docstring says `Implement this method to read data from the pipe.`. That's wrong; you implement `wrtieTo` to write data.